index/scorch/merge.go

```diff
@@ -82,7 +83,36 @@ OUTER:
 	s.asyncTasks.Done()
 }
 
-func (s *Scorch) planMergeAtSnapshot(ourSnapshot *IndexSnapshot) error {
+func (s *Scorch) parseMergePlannerOptions() *mergeplan.MergePlanOptions {
+	mergePlannerOptions := &mergeplan.DefaultMergePlanOptions
+	scorchOptions := map[string]interface{}{}
+	if v, ok := s.config["scorchOptions"]; ok {
```

Comment:
ah, a naming thing!  Instead of "scorchOptions", perhaps this is just "scorchMergePlanOptions", so it's really specific.  If one future day we add even more "scorchOptions" that aren't related to merge planner options, they won't need to be mixed up together.